README.md

```diff
@@ -18,10 +18,10 @@ An example of using the custom picklist message widget would be displaying a cus
 
 ## Release Notes
 
-Following enhancements have been made to the custom picklist message widget in version 1.1.0:
+Following enhancements have been made to the custom picklist message widget in version 2.0.0:
 
-- Added an option of displaying a spinner on the detail view of the record for the duration of completion of the task mentioned in the custom widget.
-- Added an option of displaying an icon associated with the custom message on the detail view of the record. 
+- Added an option to add timer to change the custom message.
```

Comment:
Verify readme content from Namrata Mam once.

Reply:
@cs-utkarsh-surana , please add more details around this feature wrt user usability